minesweeper.py

```diff
@@ -46,21 +46,20 @@ def mines_around_square(self, coordinate):
         """Looks at the squares adjacent to current_square and counts
         how many mines there are"""
         adjacent_mines = 0
-        for mine in self.mine_locations:
-            if (mine[0] == coordinate[0] - 1 or mine[0] == coordinate[0] + 1) and (
-                mine[1] == coordinate[1]
-                or mine[1] == coordinate[1] - 1
-                or mine[1] == coordinate[1] + 1
-            ):
-                adjacent_mines += 1
-            elif (mine[1] == coordinate[1] - 1 or mine[1] == coordinate[1] + 1) and (
-                mine[0] == coordinate[0]
-                or mine[0] == coordinate[0] - 1
-                or mine[0] == coordinate[0] + 1
-            ):
-                adjacent_mines += 1
+        x,y = coordinate
+        for mine_x, mine_y in self.mine_locations:
```

Comment:
```suggestion
        for mine in self.mine_locations:
```

There's no need to do `for mine_x, mine_y in ...` because you always use them to make a tuple `(mine_x, mine_y)`. If you do the loop like I'm suggesting here, then you will need to change `(mine_x, mine_y)` to just `mine` in the `if` checks. To then split it on multiple lines, you can put the whole condition into parentheses, like this:

```python
if (mine == (x, y-1) or
    mine == (x, y+1) or
    ...
```

Or you can just write it on one long line and run `black minesweeper.py` to rewrite it nicely (see README).